src/styles.ts

```diff
@@ -1,5 +1,5 @@
-import { css, cx } from 'emotion';
-import { Interpolation } from 'create-emotion';
+import { css, cx } from '@emotion/css';
+import { CSSInterpolation } from '@emotion/serialize';
```

Comment:
`CSSInterpolation` is also included in `@emotion/css`, so we shouldn't need to bring in another package just for this type. 
```suggestion
import { css, CSSInterpolation, cx } from '@emotion/css';
```